modules/move-etcd-different-disk.adoc

```diff
@@ -50,146 +82,91 @@ metadata:
+            ExecStart=/usr/sbin/setsebool -P rsync_full_access 1
+            ExecStart=/bin/rsync -ar /sysroot/ostree/deploy/rhcos/var/lib/etcd/ /var/lib/etcd/
+            ExecStart=/usr/sbin/semanage fcontext -a -t container_var_lib_t '/var/lib/etcd(/.*)?'
+            ExecStart=/usr/sbin/setsebool -P rsync_full_access 0
+            TimeoutSec=0
+
+            [Install]
+            WantedBy=multi-user.target graphical.target
+        - name: restorecon-var-lib-etcd.service
+          enabled: true
+          contents: |
+            [Unit]
+            Description=Restore recursive SELinux security contexts
+            DefaultDependencies=no
+            After=var-lib-etcd.mount
+            Before=crio.service
+
+            [Service]
+            Type=oneshot
+            RemainAfterExit=yes
+            ExecStart=/sbin/restorecon -R /var/lib/etcd/
+            TimeoutSec=0
+
+            [Install]
+            WantedBy=multi-user.target graphical.target
 ----
 
-. Log in to the cluster as a user with `cluster-admin` privileges and create the machine configuration:
-+
-[source,terminal]
-----
-$ oc login -u <username> -p <password>
-----
-+
-[source,terminal]
-----
-$ oc create -f etcd-mc.yml
-----
-+
-The nodes are updated and rebooted. After the reboot completes, the following events occur:
-+
-* An XFS file system is created on the specified disk.
-* The disk mounts to `/var/lib/etcd`.
-* The content from `/sysroot/ostree/deploy/rhcos/var/lib/etcd` syncs to `/var/lib/etcd`.
-* A restore of `SELinux` labels is forced for `/var/lib/etcd`.
-* The old content is not removed.
-
-. After the nodes are on a separate disk, update the `etcd-mc.yml` file with contents such as the following, replacing instances of `<new_disk_name>` with the noted device name:
-+
-[source,yaml]
-----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfig
-metadata:
-  labels:
-    machineconfiguration.openshift.io/role: master
-  name: 98-var-lib-etcd
-spec:
-  config:
-    ignition:
-      version: 3.4.0
-    systemd:
-      units:
-      - contents: |
-          [Unit]
-          Description=Mount /dev/<new_disk_name> to /var/lib/etcd
-          Before=local-fs.target
-          After=var.mount
-
-          [Mount]
-          What=/dev/<new_disk_name>
-          Where=/var/lib/etcd
-          Type=xfs
-          Options=defaults,prjquota
-
-          [Install]
-          WantedBy=local-fs.target
-        enabled: true
-        name: var-lib-etcd.mount
-----
-
-. Apply the modified version that removes the logic for creating and syncing the device to prevent the nodes from rebooting:
-+
-[source,terminal]
-----
-$ oc replace -f etcd-mc.yml
-----
 
 .Verification steps
 
-* Run the `grep <new_disk_name> /proc/mounts` command in a debug shell for the node to ensure that the disk mounted:
+* Run the `grep /var/lib/etcd /proc/mounts` command in a debug shell for the node to ensure that the disk is mounted:
```

Comment:
Consistency: You replaced "in a debug shell" with "at a debug shell prompt" earlier in the document.